scripts/confirmJenkinsConfigurations.sh

```diff
@@ -0,0 +1,43 @@
+set -e
+
+if [ "$#" -ne 4 ]; then
+    echo "This script is used during the SetupReleaseConfig step in Jenkins. It confirms correct configurations that will be used by Jenkins during Reactome's data release."
+    echo ""
+    echo "Usage: 'bash confirmJenkinsConfigurations.sh configFilepath jenkinsEnvironmentReleaseNumber jenkinsEnvironmentPreviousReleaseNumber userInputReleaseNumber'"
+    exit 1
+fi
+
+configFile=$1
+environmentalReleaseNumber=$2
+environmentalPrevReleaseNumber=$3
+userInputReleaseNumber=$4
+
+configFileReleaseNumber=
+
+while read line; do
+  if [[ $line =~ releaseNumber= ]] ; then
+    configFileReleaseNumber=${line#*=}
+  fi
+done < $configFile
+
+if [[ $userInputReleaseNumber != $environmentalReleaseNumber ]] ; then #|| [[ $userInputReleaseNumber != $configFileReleaseNumber ]]; then
+  echo "User input release number ($userInputReleaseNumber) does not match Jenkins environmental release number ($environmentalReleaseNumber)."
+  echo "Please update release-specific environmental variables found at 'Jenkins -> Manage Jenkins -> Configure System -> Global Properties'."
+  exit 2
```

Comment:
Just curious: why an exit value of 2?

Reply:
I'm not especially familiar with exit numbers actually, all I needed to happen was for Jenkins to see that it exited due to incorrect behaviour (see: error). Since it was an error of my own description, I was just looking for basic 'exit with errors' behaviour, and this worked

Reply:
I'm not too familiar with exit codes either, but this looks like a good, quick summary:
http://www.tldp.org/LDP/abs/html/exitcodes.html

Reply:
Thanks, I changed all exit codes to 1